src/os/ObjectStore.h

```diff
@@ -1550,7 +1550,7 @@ class ObjectStore {
     return false;   // assume a backend cannot, unless it says otherwise
   }
 
-  virtual int statfs(struct store_statfs_t *buf) = 0;
+  virtual int statfs(struct store_statfs_t *buf, osd_alert_list_t& alerts) = 0;
```

Comment:
let's do osd_alert_list_t *alerts since it's an output argument

Reply:
could also make it default to nullptr and make it optional if we want to avoid so much interface churn